ai/STA/Strategy/prepare_kickoff_offense.py

```diff
@@ -0,0 +1,71 @@
+        defense_right = self.game_state.my_team.available_players[2]
+        defense_left = self.game_state.my_team.available_players[4]
+        goalkeeper = self.game_state.my_team.available_players[5]
+
+
+        # Positions objectifs des joueurs
+        attack_left_position = Pose(Position(self.game_state.const["FIELD_X_LEFT"] / 15,
+                                                    self.game_state.const["FIELD_Y_BOTTOM"] * 3 / 5))
+        attack_right_position = Pose(Position(self.game_state.const["FIELD_X_LEFT"] / 15,
+                                                    self.game_state.const["FIELD_Y_TOP"] * 3 / 5))
+        middle_position = Pose(Position(self.game_state.const["FIELD_X_LEFT"] / 30, 0))
+        defense_right_position = Pose(Position(self.game_state.const["FIELD_X_LEFT"] / 2,
+                                                    self.game_state.const["FIELD_Y_BOTTOM"] / 3))
+        defense_left_position = Pose(Position(self.game_state.const["FIELD_X_LEFT"] / 2,
+                                                    self.game_state.const["FIELD_Y_TOP"] / 3))
+        goalkeeper_position = Pose(Position(self.game_state.const["FIELD_X_LEFT"] * 9/10, 0))
+
+
+        # Tactiques
+        self.add_tactic(attack_right.id, GoToPositionPathfinder(self.game_state, attack_right, attack_right_position))
+        self.add_tactic(attack_right.id, Stop(self.game_state, attack_right))
+        self.add_condition(attack_right.id, 0, 1, partial(self.arrived_to_position, attack_right))
+
+        self.add_tactic(attack_left.id, GoToPositionPathfinder(self.game_state, attack_left, attack_left_position))
+        self.add_tactic(attack_left.id, Stop(self.game_state, attack_left))
+        self.add_condition(attack_left.id, 0, 1, partial(self.arrived_to_position, attack_left))
+
+        self.add_tactic(middle.id, GoToPositionPathfinder(self.game_state, middle, middle_position))
+        self.add_tactic(middle.id, Stop(self.game_state, middle))
+        self.add_condition(middle.id, 0, 1, partial(self.arrived_to_position, middle))
+
+        self.add_tactic(defense_right.id, GoToPositionPathfinder(self.game_state, defense_right, defense_right_position))
+        self.add_tactic(defense_right.id, Stop(self.game_state, defense_right))
+        self.add_condition(defense_right.id, 0, 1, partial(self.arrived_to_position, defense_right))
+
+        self.add_tactic(defense_left.id, GoToPositionPathfinder(self.game_state, defense_left, defense_left_position))
+        self.add_tactic(defense_left.id, Stop(self.game_state, defense_left))
+        self.add_condition(defense_left.id, 0, 1, partial(self.arrived_to_position, defense_left))
+
+        self.add_tactic(goalkeeper.id, GoToPositionPathfinder(self.game_state, goalkeeper, goalkeeper_position))
+        self.add_tactic(goalkeeper.id, Stop(self.game_state, goalkeeper))
+        self.add_condition(goalkeeper.id, 0, 1, partial(self.arrived_to_position, goalkeeper))
+
```

Comment:
All the tactic assignation could be change by a list. Since they all follow the template:
```
 self.add_tactic(<player>.id, <tactic>(self.game_state, <player>, <player_position>))
 self.add_tactic(<player>.id, Stop(self.game_state, <player>))
 self.add_condition(<player>.id, 0, 1, partial(self.arrived_to_position, <player>))
```
We could have a list in this style:
```
assignation = [ ( attack_right, GoToPositionPathfinder.__class__, attack_right_position),
                        (attack_left, GoToPositionPathfinder.__class__, attack_left_position),
                        ... ]
for player, tactic_class, player_position in assignation:
      self.add_tactic(player.id, tactic_class.__init__(self.game_state, player, player_position))
      ...
```
I am not sure if we can use __init__ to construct from a class type.